client/src/components/admin/VerificationRequests.tsx

```
@@ -1,83 +1,135 @@
-        </table>
-      </div>
+            </thead>
+            <tbody>
+              {requests.map((req) => (
+                <tr key={req._id} className="border-t">
+                  <td className="px-4 py-3 text-sm text-gray-500">{req.user}</td>
+                  <td className="px-4 py-3">
+                    <button
+                      className="text-blue-600 font-semibold hover:underline text-xs md:text-sm"
+                      onClick={() => setViewImageUrl(req.idCardImage)}
+                    >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)